src/meta_schedule/schedule_rule/multi_level_tiling_tensor_core.cc

```diff
@@ -529,7 +529,7 @@ inline std::vector<State> MultiLevelTilingTensorCoreNode::TransformForTensorizat
                        state->intrin_group.compute_intrin);
   state->sch->Annotate(state->block_rv, tir::attr::meta_schedule_auto_tensorize_init,
                        state->intrin_group.init_intrin);
-  state->sch->Annotate(state->block_rv, tir::attr::warp_execution, Bool(true));
```

Comment:
A quick question: Can we keep the `Bool(true)`? Since it is more clear to indicate it's a bool value

Reply:
Bool is not compatible with our FFI, so I got it changed